docs/reference/common-xref.md

```diff
@@ -0,0 +1,31 @@
+# Common dbxref and their shorthands
+
+This is a list of commonly used xrefs shorthands. For full list, please refer to https://github.com/geneontology/go-site/blob/master/metadata/db-xrefs.yaml
+
+| Name                                                     | dbxref shorthand    | Example                                            | Expanded                                                      |
+| -------------------------------------------------------- | ------------------- | -------------------------------------------------- | ------------------------------------------------------------- |
+| bioRxiv                                                  | bioRxiv:            | bioRxiv:10.1101/530881v2                           | https://www.biorxiv.org/                                      |
+| Digital Object Identifier                                | DOI:                | DOI:10.1016/S0963-9969(99)00021-6                  | http://dx.doi.org/                                            |
+| Evidence Code Ontology                                   | ECO:                | ECO:0000315                                        | http://www.evidenceontology.org/browse/#ECO\_                 |
```

Comment:
Is this somehow different from https://www.ebi.ac.uk/ols/ontologies/eco? If so, we should use the proper expansion to http://purl.obolibrary.org/obo/ECO_

Reply:
argh didnt notice the `\` -> happened through converting excel to md 
but anyway, I copied it from https://github.com/geneontology/go-site/blob/master/metadata/db-xrefs.yaml (wanted to be accurate about the redirect) so if there needs to be a change, I guess we need to make a request in GO? (or we make our own redirect, which I think @dosumis would want)

Reply:
If this table is prescriptive, lets perhaps just add rows here for prefixes we really (really) use in our curation efforts and define the expansions that are right. GO db-xrefs needs to be replaced by something more modern.

Reply:
I think @dosumis is right that we should just link to yaml file for now (not the nicest looking thing) and replace once we have better ways of doing it